client/sections.js

```diff
@@ -108,6 +108,13 @@ const sections = [
 		module: 'calypso/my-sites/plugins',
 		group: 'sites',
 	},
+	{
+		// TODO: Move this out to its own root folder
+		name: 'marketplace',
```

Comment:
Is there a more specific name we could call this section? Is it a plugin marketplace, or a domain marketplace? I think it would be useful to reflect that in the naming just to avoid potential confusion with any other marketplaces that exist or will exist in the future 😉

Reply:
Having the name plugin-marketplace can be confusing since we already have a "plugins" place to add plugins. Also we bundle upsell a domain here. So the generic marketplace is the right term for it, at least right now.

Aspirations for an elaborate marketplace are far into the future and we agreed to not let any nuances of the "marketplace" effect our technical decisions. Basically the instructions were to go with a separate "Yoast Sale" page if required. So we can live with this name for now and allowances and analysis will be made when we truly want this to be a generic marketplace or specialised one. I went with this names so that we are making at least some sort of allowances in our code for it and to make sure the user would not be confused by a route like wordpress.com/yoast.

Reply:
Sounds good to me, and this is a product decision so I'm totally fine with whatever you'd like to go with. Just wanted to get your thoughts on it 👍